k8s/manifest-templates/pharos.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pharos
+  labels:
+    name: pharos
+spec:
+  hostNetwork: true
+  containers:
+  - name: pharos-dashboard
+    image: ereslibre/pharos-dev:latest
```

Comment:
It should be the same, building the image of fetching it unless someone has to change the Dockerfile for some reason in branch. In that case, we would build it locally and work with it as long as the branch is not merged and then we would push it to the registry. I suggest we use the ```imagePullPolicy: Never``` option to avoid confusion when the ```latest``` tag is used ([link](https://kubernetes.io/docs/user-guide/pods/multi-container/#containers)).
also.